clients/vscode-hlasmplugin/schema/asm_options.schema.json

```diff
@@ -42,7 +42,7 @@
         },
         "PROFILE": {
             "type": "string",
-            "description": "Profile Member to be copied into the source program."
+            "description": "Profile Member that is copied into the source program."

```

Comment:
Why?

Reply:
it describes what the plugin will do, not the present state of things.

Reply:
In tech writing, we don't use the Future tense - again, a standard. Things are happening presently. 
Unless really necessary, you can use it ofc, but we try to avoid that.

So, in this case, the wording would be "Profile Member that will be copied into the source program."